.github/workflows/ci.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.11']
-        toxenv: [quality, docs, django42-drflatest]
+        python-version: ['3.12']
+        toxenv: [quality, docs, django42-drflatest, django42]
```

Comment:
We already have `django42-drflatest` target so we don't need to add another `django42` target.